### DISCUSSION TITLE
Welcome to Awesome-CloudOps-Automation Discussions!

### DISCUSSION THREAD

**Discussion by JBAhire** [unskript/Awesome-CloudOps-Automation#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Other than that, 

- 🐞 If you found a bug, it is best to [submit an issue](https://github.com/unskript/Awesome-CloudOps-Automation/issues/new) to ensure our team resolves your issue.
- ❓ If you have a question, search in existing discussions or [join us on slack](https://join.slack.com/t/cloud-ops-community/shared_invite/zt-1fvuobp10-~r_KyK9BxPhGiebOvl3h_w) to reach out to our team. 
-  🆕 Otherwise, start a new discussion.

**Comment by hemanthpatil30** [comment]
Hello, I am new to unskript. I have setup this locally and trying to find a way for the following questions. guide or share documentation on the questions.

I have tried to setup with unskript/awesome-runbooks:latest or unskript/awesome-runbooks:1.20.0

1. Steps to setup open source in kubernetes
2. Runbook is failing for creation of AWS IAM user, gives certificate error or fails at "Create login profile for IAM user"
3. Unable to setup AWS profile as its failing to import kubernetes config file.
4. How to store credentials and parameters in volumes in kubernetes setup.

**Comment by jayasimha-raghavan-unskript** [comment]
Hi There!

Great to see your interest and thanks for trying Awesome Runbooks out!  
1. Setup Kubernetes:  Start here: https://kubernetes.io/docs/tasks/tools/
2. Yes, Runbooks would need `Credential` In this case, your Runbook, specifically AWS Runbook need a AWS Credential (Role ARN or Secret/access key)
3. Did not understand this, As long as you have `AWS_PROFILE` or `KUBECONFIG` Env variables set (and are valid), you can just import them.  Typically when you launch the Docker, you can use the `-v` option to mount your local host's `.aws` folder onto the docker instance.
4. If you are using docker, you can mount a local directory using `-v` and launch the docker, this will store the credentials on that local directory. Look at this document https://github.com/unskript/Awesome-CloudOps-Automation/blob/master/README_extending_docker.md for more information.
